communities.json

```diff
@@ -771,6 +771,14 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "bentobox.png",
+      "title": "BentoBoxWorld",
+      "quote": "Something you want to know? Ask us a question in the Discussions or on our Discord server. We will be glad to help you.",
+      "quoteSourceUrl": "https://www.spigotmc.org/resources/bentobox-bskyblock-acidisland-skygrid-caveblock-aoneblock.73261/",
+      "inviteCode": "bentobox",
```

Comment:
It looks like this invite code doesn't currently exist. I cannot identify your server without a valid invite code. You could also send me your server ID. https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-

Reply:
Shall I update it in the PR? The current invite is [ur8EPYS](https://discord.com/invite/ur8EPYS)